VTEX - Antifraud Provider API.json

```diff
@@ -984,60 +631,29 @@
         "/transactions/{transactions.id}": {
             "get": {
                 "tags": [
-                    "Antifraud  Flow"
+                    "Anti-fraud Flow"
                 ],
-                "summary": "Get Antifraud Status",
-                "description": "This endpoint is used to get the antifraud analysis status. This request is performed every two hours up to 5 days. After 5 days in `undefined` status, the order will expire and be cancelled.",
+                "summary": "Get Anti-fraud Status",
+                "description": "Retrieves the anti-fraud analysis status. This request is performed every two hours up to 5 days. After 5 days in `undefined` status, the order will expire and be cancelled.\r\n\r\n## Permissions\r\n\r\nThis endpoint does not require [authentication](https:\/\/developers.vtex.com\/docs\/guides\/authentication) or [permissions](https:\/\/help.vtex.com\/en\/tutorial\/license-manager-resources--3q6ztrC8YynQf6rdc6euk3).",
```

Comment:
Se esse endpoint não requer authentication, por que tem security nele?

Reply:
Adicionado nova descrição, conforme conversamos: ## Permissions\r\n\r\nThis endpoint does not require [permissions](https:\/\/help.vtex.com\/en\/tutorial\/license-manager-resources--3q6ztrC8YynQf6rdc6euk3).